source/users/custom-metadata.txt

```diff
@@ -71,10 +76,10 @@ When you use custom user data, keep the following things in mind:
 
 - **Custom Data May Be Stale**: A user's custom data is *sourced* from a
   MongoDB collection but is stored in and read from a user's
-  authentication access token. If a user has a valid access token when
-  the underlying document changes, their custom data in that session
-  does not update until they refresh their access token or
-  re-authenticate.
+  authentication access token. If a user's metadata is updated after the 
+  access token is issued, requests using the previously created access 
+  token will not have the newly updated metadata. Their user metadata 
+  will update when they refresh their access token or re-authenticate.
```

Comment:
[Custom user data](https://www.mongodb.com/docs/atlas/app-services/users/custom-metadata/#custom-user-data) and [user metadata from authentication providers](https://www.mongodb.com/docs/atlas/app-services/users/custom-metadata/#authentication-provider-metadata) are two separate things so we would _not_ want to overwrite this. Instead we would want to just add this section you wrote most probably to [this section](https://www.mongodb.com/docs/atlas/app-services/users/custom-metadata/#authentication-provider-metadata)